HelloSignalR/Connections/SimpleConnection.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNet.SignalR;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.Extensions.Logging;
+
+namespace HelloSignalR.Connections
+{
+    public class SimpleConnection : PersistentConnection
+    {
+        protected override async Task OnConnected(HttpRequest request, string connectionId)
+        {
+            await base.OnConnected(request, connectionId);
```

Comment:
FYI, calling `base.OnConnected` is not useful as it just returns a completed task: https://github.com/aspnet/SignalR-Server/blob/dev/src/Microsoft.AspNet.SignalR.Server/PersistentConnection.cs#L431